tests/constraint_tests.py

```diff
@@ -225,6 +225,7 @@ def test_storage(self):
             outputs={bel: solph.Flow(nominal_value=16667, variable_costs=24)},
             nominal_storage_capacity=10e4,
             loss_rate=0.13,
+            fixed_losses=0.01,
```

Comment:
Please add a new test instead of changing an existing one.

Reply:
I reverted the commit and will add a new test.